assets/sass/components/_past-grants.scss

```diff
@@ -60,9 +60,12 @@
         }
 
         &.is-ascending::before,
-        &.is-ascending::after {
+        &.is-descending::before {
             position: relative;
             top: 2px; // Optically align
+            margin-right: 4px;
+            display: inline-block;
```

Comment:
Added this to fix this:

![image](https://user-images.githubusercontent.com/394376/46604465-44cb4700-caee-11e8-90d4-281b60ca29fc.png)